common/engine_conf/parser.cc

```diff
@@ -703,7 +706,8 @@ void parser::_merge(std::unique_ptr<message_helper>& msg_helper,
         } else {
           switch (f->cpp_type()) {
             case FieldDescriptor::CPPTYPE_STRING:
-              refl->SetString(msg, f, refl->GetString(*tmpl, f));
+              if (!(desc->name() == "Host" && f->name() == "alias"))
+                refl->SetString(msg, f, refl->GetString(*tmpl, f));
               break;
```

Comment:
wrong place...
the parser must be generic, it works with all types of object.

Reply:
will fix